src/selectors/prescription.js

```diff
@@ -71,7 +71,7 @@ export const selectFilteredAndSortedItems = createSelector(
 
 export const selectSelectedRows = ({ prescription }) => {
   const { transaction } = prescription;
-  const { items } = transaction;
+  const { items = [] } = transaction || {};
 
   return items.reduce((acc, { item }) => ({ ...acc, [item.id]: true }), {});
```

Comment:
Nit: I think your way more elegant, but I think a little bit more readable to do:

```
const { items } = transaction || {};
return items?.reduce(...) ?? []; 
```